### PR TITLE
cls migration

### PR DESCRIPTION
**Pull request by SteveDMurphy**
Closes [fidesctl-plus#104](https://github.com/ethyca/fidesctl-plus/issues/104)

### Code Changes

* [x] New migration to manage classification state

### Steps to Confirm

* [x] The classification models are functional as is across projects

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation Updated:
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [x] documentation issue created (tag docs-team to complete issue separately)
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`

### Description Of Changes

This is slightly complicated to validate but shouldn't be too problematic. This migration is more for base functionality with some planning ahead for the future, however it is understood that some future changes are highly likely to these tables.

### CODE REVIEW

File: src/fidesctl/api/ctl/sql_models.py [hunk 1]
```diff
@@ -85,6 +86,67 @@ def process_result_value(self, value, dialect):
...
+
+
+class ClassificationState(Base):
+    """
+    The SQL model for a classification instance
+    """
+
+    __tablename__ = "cls_classification_status"
+
+    classification_status_id = Column(Integer, primary_key=True, autoincrement=True)
```
Comment (SteveDMurphy) [inline src/fidesctl/api/ctl/sql_models.py:139]: This is easily confused with the above `classification_status_id`

File: src/fidesctl/api/ctl/migrations/versions/69d3ce8a040e_classification_management.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+down_revision = "4fc34906c389"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "cls_classification_detail",
+        sa.Column("id", sa.String(length=255), nullable=False),
```
Comment (SteveDMurphy) [inline src/fidesctl/api/ctl/migrations/versions/69d3ce8a040e_classification_management.py:22]: Look at best practices for how to make this performant and consistent with naming

File: src/fidesctl/api/ctl/migrations/versions/69d3ce8a040e_classification_management.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        sa.Column(
+            "classification_detail_id", sa.Integer(), autoincrement=True, nullable=False
```
Comment (SteveDMurphy) [inline src/fidesctl/api/ctl/migrations/versions/69d3ce8a040e_classification_management.py:24]: Also look at dropping classification from the name, it's a bit repetitive (and unnecessary)

File: src/fidesctl/api/ctl/sql_models.py [hunk 4]
```diff
@@ -85,6 +86,46 @@ def process_result_value(self, value, dialect):
...
 
 
+class ClassificationDetail(Base):
+    """
+    The SQL model for a classification instance
+    """
+
+    __tablename__ = "cls_classification_detail"
+    instance_id = Column(String(255))
+    status = Column(Text)
```
Comment (SteveDMurphy) [inline src/fidesctl/api/ctl/sql_models.py:96]: Provide a more explicit name for what status this is for (i.e. `review_status`)

Comment (SteveDMurphy) [reply src/fidesctl/api/ctl/sql_models.py:95]: As an alternative solution here, maybe we can provide better context of which status this means (detail field level) with the API responses regardless of column name here

**Comment by SteveDMurphy** [comment]
This needs to have the migrations updated to reflect #1103

**Review by ThomasLaPiana — APPROVED** [review]
LGTM!

**Pull request merged**